doc/website/getting-started/what-is-iceoryx.md

```diff
@@ -1 +1,24 @@
+
+
+## Fast
+With the iceoryx API, a publisher can write the message directly into a chunk of memory that was previously requested from the middleware. When the message is delivered, subscribers receive reference counted pointers to these memory chunks, which are stored in queues with configurable capacities. With this we achieve what we refer to as true zero-copy — an end-to-end approach from publishers to subscribers without creating a single copy.
+
+Avoiding the copies on API level is crucial when GBytes of sensor data have to be processed per second on robotics and autonomous driving systems. Therefore the iceoryx team contributed to the standardization of true zero-copy capable APIs in [ROS 2](https://www.ros.org/) and [AUTOSAR Adaptive](https://www.autosar.org/standards/adaptive-platform/). 
+  
+On modern processors iceoryx has a latency of less than 1 µs for transferring a message. And the best message is that this latency is constant as size doesn't matter. Want to give it a try? Then have a look at our [iceperf example](../examples/iceperf) after having made the first steps. 
+## Flexible
+iceoryx already supports Linux, QNX and MacOS as operating systems as well as C and C++ as user APIs. Windows and Rust are the next ones on the list. The typed C++ API is the most comfortable when you want to go for the iceoryx API on the user side. The untyped C++ API and the C API provide a data agnostic interface that is often preferred when integrating iceoryx as shared memory backbone into a bigger framework.
+
+The APIs support polling access and event-driven interactions with the [Waitset](../overview/#waitset) and [Listener](../overview/#listener). Applications can be started and stopped flexibly as there is a service discovery behind the scenes that dynamically connects matching communication entities. 
+
+That iceoryx has the right set of features can be seen from the already existing integrations in middleware and frameworks such as [Eclipse Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds), [eCAL from Continental](https://continental.github.io/ecal/), [RTA-VRTE from ETAS](https://www.etas.com/en/products/rta-vrte.php) and [Apex.OS from Apex.AI](https://www.apex.ai/apex-os).  
+## Dependable
+The predecessors of iceoryx are running in millions of vehicles worldwide. We know the requirements coming from the safety critical automotive domain and have these in mind for the features we design and implement. We avoid the usage of heap, exceptions and any undefined behavior to increase the predictability. Instead we are using our own memory allocation based on static memory pools and we were inspired by upcoming C++ features and other languages like Rust for how to deal with return values and error cases (details can be found [here](../../advanced/how-optional-and-error-values-are-returned-in-iceoryx/)).
```

Comment:
```suggestion
The predecessor of iceoryx is running in millions of vehicles worldwide. All iceoryx maintainers hail from the safety critical automotive domain. Hence, they know the necessary requirements and have these in mind for the design and implement of the features. The usage of heap, exceptions and any undefined behavior are to be avoided in iceoryx to increase the predictability. Instead a custom memory allocation is being used, based on static memory pools and was inspired by upcoming C++17/20 features and other languages like Rust. For how to deal with return values and error cases (details can be found [here](../../advanced/how-optional-and-error-values-are-returned-in-iceoryx/)).
```

You could also think about mentioning the [predecessor](https://www.autosar.org/fileadmin/user_upload/standards/adaptive/17-03/AUTOSAR_EXP_ARAComAPI.pdf)